ws-nextjs-app/pages/[service]/send/[id]/FormContext/index.tsx

```diff
@@ -124,9 +126,9 @@ export const FormContextProvider = ({
 
       if (fieldValue === '') return;
       if (isFileHtmlType) {
-        const fileList = fieldValue as File[];
+        const fileList = fieldValue as FileData[];
 
-        fileList.forEach(file => {
+        fileList.forEach(({ file }: FileData) => {
```

Comment:
May be one to check with @HarveyPeachey that this is ok to do as I remember there were some issues transforming/submitting the data when using FileData? Could be wrong though.

Reply:
I think this is fine, it was the [FileList](https://developer.mozilla.org/en-US/docs/Web/API/FileList) data type which was the issue. This looks like custom object with both the file and the error stuff which is being de-structured, so looks good to me